src/test/java/net/sf/jabref/logic/util/strings/StringUtilTest.java

```diff
@@ -186,4 +186,41 @@ public void testBooleanToBinaryString() {
         assertEquals("1", StringUtil.booleanToBinaryString(true));
     }
 
+    @Test
+    public void testIntValueOf() {
```

Comment:
Can you rename and split the test please. For example, `InfValueOfParses Small/Large NumberCorrectly`.